cmd/gardener-apiserver/app/gardener_apiserver.go

```diff
@@ -263,6 +267,25 @@ func (o Options) run(stopCh <-chan struct{}) error {
 		return err
 	}
 
+	if err := server.GenericAPIServer.AddPostStartHook("bootstrap-gardener-cluster", func(context genericapiserver.PostStartHookContext) error {
+		if _, err := kubeClient.CoreV1().Namespaces().Get(gardencorev1beta1.GardenerSeedLeaseNamespace, metav1.GetOptions{}); err == nil {
+			// the namespace already exists
+			return nil
+		}
+
+		_, err = kubeClient.CoreV1().Namespaces().Create(&corev1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: gardencorev1beta1.GardenerSeedLeaseNamespace,
+			},
+		})
+		if err != nil && !apierrors.IsAlreadyExists(err) {
+			return err
+		}
+		return nil
```

Comment:
first handle error case properly

```suggestion
		if _, err := kubeClient.CoreV1().Namespaces().Get(gardencorev1beta1.GardenerSeedLeaseNamespace, metav1.GetOptions{}); client.IgnoreNotFound(err) != nil {
			return err
		}
		if apierrors.IsNotFound(err) {
			// the namespace already exists
			return nil
		}

		_, err = kubeClient.CoreV1().Namespaces().Create(&corev1.Namespace{
			ObjectMeta: metav1.ObjectMeta{
				Name: gardencorev1beta1.GardenerSeedLeaseNamespace,
			},
		})
		return err
```